staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go

```diff
@@ -137,9 +140,15 @@ func (f *ConfigFlags) toRawKubeConfigLoader() clientcmd.ClientConfig {
 	if f.CertFile != nil {
 		overrides.AuthInfo.ClientCertificate = *f.CertFile
 	}
+	if f.CertData != nil {
```

Comment:
Can you explain me a bit how do you envision this being exposed for CLI, the idea of this is to be able to inject the values specified through the flags. If you care about directly modifying the contents of the client-go config file, that's the resource you should be using. This isn't the right place.

Reply:
It's not really to use via the command line. But the `RESTClientGetter` interface is not only being used in CLI, and sometimes the kubeconfig file is not available and, for instance, the CA is only in memeory and thus can't be used to construct a ConfigFlag. I reckon there is no direct usage from a cli flag point of view, but it allows easier construction of `RESTClientGetter` when the certs are only in memory

Reply:
But yeah totally understand that it's not really useful regarding the cli 🙄